sqs/src/main/scala/awscala/sqs/SQS.scala

```diff
@@ -39,7 +39,9 @@ trait SQS extends aws.AmazonSQS {
 
   def queues: Seq[Queue] = listQueues().getQueueUrls.asScala.map(url => Queue(url)).toSeq
 
-  def queue(name: String): Option[Queue] = queues.find(_.url.split("/").last == name)
+  def queuesByName(name: String): Seq[Queue] = listQueues(name).getQueueUrls.asScala.map(url => Queue(url)).toSeq
```

Comment:
I'd like to rename this method as `queuesByNamePrefix`. Are you still interested in modifying this PR? Otherwise, I will merge this and will modify it later.
https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/sqs/AmazonSQS.html#listQueues-java.lang.String-

Reply:
```suggestion
  def queuesByNamePrefix(name: String): Seq[Queue] = listQueues(name).getQueueUrls.asScala.map(url => Queue(url)).toSeq
```